src/index.ts

```diff
@@ -393,6 +395,14 @@ export class UnleashClient extends TinyEmitter {
         this.metrics.stop();
     }
 
+    public isReady(): boolean {
+        return this.readyEventEmitted;
+    }
+
+    public getError(): SdkState {
```

Comment:
can we rename to "getLastError"?

Reply:
It has the logic to not return it when current SDK state is not an error. `getLastError` in my mind indicates that it returns last error even if SDK already recovered

Reply:
ok, thanks for clarifying. makes sense.